src/gui/widgets/scroll_text.cpp

```diff
@@ -134,8 +135,9 @@ void scroll_text::finalize_subclass()
 
 void scroll_text::place(const point& origin, const point& size) {
 	scrollbar_container::place(origin, size);
+	multiline_text* widget = get_internal_text_box();
 
-	if(multiline_text* widget = get_internal_text_box()) {
+	if(widget) {
```

Comment:
This can go back to initializing widget in the condition of the `if`, as widget isn't accessed outside the `if`.

Trivial, but as this needs another push to pass the CI's whitespace check it might as well have this change too.

Reply:
Updated.